pallets/storage-provider/src/lib.rs

```diff
@@ -388,6 +386,8 @@ pub mod pallet {
 
     #[pallet::call]
     impl<T: Config> Pallet<T> {
+        #[pallet::call_index(0)]
```

Comment:
If we just benchmarked this crate to get proper weights, why are we setting weights to 0?
If we don't have proper weights for every extrinsic yet, let's just leave it be.